examples/zkapps/04-zkapp-browser-ui/ui/pages/_app.page.tsx

```diff
@@ -59,7 +59,7 @@ export default function App() {
         console.log('zkApp compiled');
 
         const zkappPublicKey = PublicKey.fromBase58(
-          'B62qph2VodgSo5NKn9gZta5BHNxppgZMDUihf1g7mXreL4uPJFXDGDA'
```

Comment:
@ymekuria Is this intentional to be included?

Reply:
@jasongitmail, I added an updated zkApp public key of a deployed contract for this tutorial. It is unrelated to adding notes to each tutorial and can be removed here and added to another PR.

Reply:
Ok, for expediency will merge